### PR TITLE
defect/cybtachyon/DAT-916--json-library

### PR DESCRIPTION
**Pull request by cybtachyon**

### CODE REVIEW

File: src/Plugin/Block/PatternkitBlock.php [hunk 1]
```diff
@@ -554,6 +563,12 @@ public function build() {
           $template = str_replace($token, $placeholder, $template);
         }
       }
+      // If the user is not using Twig templating,
+      // wrap with a Twig write so we can process it.
+      if (strpos($template, '{{') === FALSE
```
Comment (ryanissamson) [inline src/Plugin/Block/PatternkitBlock.php]: This seems to break for pattern templates, so would probably need more ifs to check for @ at least. I'm wondering if we could just wrap the tokens in the prior if statement at $template_context[$placeholder] = $tokenized[$token]; The $token value is already right, you'd just need to wrap $placeholder with the brackets. And as it's already in a for loop, multiple values shouldn't matter.

Comment (cybtachyon) [reply src/Plugin/Block/PatternkitBlock.php]: I'm not necessarily sure that's a valid use case. Only simple twig filters and loops are supported inline for Patternkit values, no embeds or includes would be expected.

Comment (cybtachyon) [reply src/Plugin/Block/PatternkitBlock.php]: Marking this as resolved, if we want to revisit officially supporting includes/embeds we should open a new issue.

**Comment by wats958** [comment]
Reviewed.

**Pull request merged**